tests/Http/HttpRequestTest.php

```diff
@@ -537,4 +537,22 @@ public function testCreateFromBase()
 
         $this->assertEquals($request->request->all(), $body);
     }
+
+    public function testJsonContent()
+    {
+        $body = [
+            'foo' => 'bar',
+            'baz' => ['qux'],
+        ];
+
+        $server = [
+            'CONTENT_TYPE' => 'application/json',
```

Comment:
Nope, CONTENT_TYPE is the appropriate header.

From elsewhere in the REQUEST object:

```php
/**
 * Determine if the request is sending JSON.
 *
 * @return bool
 */
public function isJson()
{
    return Str::contains($this->header('CONTENT_TYPE'), '/json');
}
```